_data/main-nav.yml

```diff
@@ -1,19 +1,19 @@
 - label: Cloud
   children:
 
-    - label: Cloud guide
+    - label: Cloud Guide
       url: /cloud/bk-cloud.html
 
-    - label: Cloud architecture
+    - label: Cloud Architecture
       url: /cloud/architecture/cloud-architecture.html
 
-    - label: Configure environments
+    - label: Configure Environments
       url: /cloud/env/environments.html
 
-    - label: Upgrades and patches
+    - label: Upgrades and Patches
       url: /cloud/project/project-upgrade-parent.html
 
-    - label: ece-tools release notes
+    - label: ece-tools Release Notes
```

Comment:
On this one consider changing to: 

Release Notes (ece-tools)  or Release Notes: ece-tools